src/hotspot/share/cds/lambdaProxyClassDictionary.cpp

```diff
@@ -23,9 +23,15 @@
  */
 
 #include "cds/archiveBuilder.hpp"
+#include "cds/cdsConfig.hpp"
+#include "cds/cdsProtectionDomain.hpp"
 #include "cds/lambdaProxyClassDictionary.hpp"
 #include "classfile/systemDictionaryShared.hpp"
+#include "interpreter/bootstrapInfo.hpp"
+#include "jfr/jfrEvents.hpp"
```

Comment:
Extra include?

Reply:
jfrEvents.hpp is needed by `EventClassLoad` on line 360